docs/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md

```diff
@@ -2,26 +2,35 @@
 
 This page provides guidance on how to manage notifications for Airbyte Cloud, allowing you to stay up-to-date on the activities in your workspace. 
 
-## Set up Webhook notifications
+
+## Configure Notification Settings
 
 To set up Webhook notifications:
 
 1. On the [Airbyte Cloud](http://cloud.airbyte.com) dashboard, click **Settings**.
 
 2. Click **Notifications**.
 
-3. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
+3. Have a webhook URL ready if you plan to use webhook notifications. Using a Slack webhook is recommended. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
+
+4. Toggle the type of events you are interested to receive notifications for. 
+	1. If using slack webhooks, the webhook URL is required. For your convenience, we provide a 'test' function to send a test message to your slack channel so you can make sure it's working as expected.
+
+5. Click **Save changes**.
 
-4. Navigate back to the Airbyte Cloud dashboard > Settings > Notifications and enter the Webhook URL.
+## Notification Event Types
 
-5. Toggle the **When sync fails** and **When sync succeeds** buttons as required.
+1. Failed syncs and successful syncs: When a connection sync has finished, you can choose to be notified whether the sync has failed, succeeded or both. Note that if sync runs frequently or if there are many syncs in the workspace these types of events can be noisy.
+1. Automated Connection Updates: when Airbyte detects that the connection's source schema has changed and can be updated automatically, Airbyte will update your connection and send you a notification message.
```

Comment:
The numbers here are all 1. Could we get those numbered in sequence?

Reply:
that's markdown format - after rendering it should look as expected - you can verify it in https://markdownlivepreview.com/

Reply:
Oh okay, makes sense!